src/semantic-ui/site/elements/list.overrides

```diff
@@ -32,6 +32,12 @@ List Overrides - REACT-INVENIO-APP-ILS
     padding: 0.5em;
   }
 
+  &.metadata-sharp > * > * > *{
```

Comment:
in terms of CSS and semantic UI combination, the parent child selectors usage is quite dangerous - if someone changes the structure, or in the new version of semantic UI we will have more elements in this hierarchy, this stylesheet will not be applied correctly. is there any existing out of the box component you could use instead of the custom CSS? (custom CSS should be used only as a last resort, if there is no other solution)